src/utils/fs.js

```diff
@@ -0,0 +1,16 @@
+const path = require("path");
+const fs = require("fs");
+
+function getPackageJson() {
+  const packageJsonPath = path.resolve(process.cwd(), "package.json");
```

Comment:
I think here we need to be careful because the `initialize` function could be run with a folder parameter in the options:
https://github.com/restqa/restqa/blob/fe02ef7f19302b96035426bbd695b893bde84528/src/cli/initialize.js#L99

So might need to be more carefull of using `process.cwd()`

Reply:
The problem is that we call `getPackageJson` before the `initialize.generate function`. I add an optional parameter anyway.